simple_model:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 20
    
  Rev5_position_controller:
    type: position_controllers/JointPositionController
    joint: Rev5
    pid: {p: 100.0, i: 0.01, d: 10.0}

  diff_drive_controller:
    type: "diff_drive_controller/DiffDriveController"
    left_wheel: ['Rev2', 'Rev1']
    right_wheel: ['Rev3', 'Rev4']
    publish_rate: 100
    pose_covariance_diagonal: [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 0.03]
    twist_covariance_diagonal: [0.001, 0.001, 0.001, 1000000.0, 1000000.0, 0.03]
    cmd_vel_timeout: 0.25

    # Odometry fused with IMU is published by robot_localization, so
    # no need to publish a TF based on encoders alone.
    enable_odom_tf: false

    # Wheel separation and radius multipliers
    wheel_separation_multiplier: 1.5 # default: 1.0
    wheel_radius_multiplier    : 1.0 # default: 1.0

    # Velocity and acceleration limits
    # Whenever a min_* is unspecified, default to -max_*
    linear:
      x:
        has_velocity_limits    : true
        max_velocity           : 2.0   # m/s
        has_acceleration_limits: true
        max_acceleration       : 20.0   # m/s^2
    angular:
      z:
        has_velocity_limits    : true
        max_velocity           : 4.0   # rad/s
        has_acceleration_limits: true
        max_acceleration       : 25.0   # rad/s^2
